typespec-tests/src/main/java/com/_specs_/azure/core/model/ModelAsyncClient.java

```diff
@@ -55,7 +55,7 @@ public final class ModelAsyncClient {
      * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
      * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
-     * @return an embedding vector along with {@link Response} on successful completion of {@link Mono}.
+     * @return the response body along with {@link Response} on successful completion of {@link Mono}.
```

Comment:
Response description related changes depends on this issue: https://github.com/Azure/typespec-azure/issues/1088. Response description should be updated once we updated the TCGC version. I will check together after upgrading.

Reply:
The response description is not updated after using `sdkResponse.description` as we already get description from raw type previously, so the result are the same. I'm checking where these diffs come from.

Reply:
ok, I get the root cause of the response description difference now.

In codegen, in [createSimpleSyncClientMethods](https://github.com/Azure/autorest.java/blob/8ee365815f616d708a129b92610e8e20f24e8b38/javagen/src/main/java/com/azure/autorest/mapper/ClientMethodMapper.java#L811), it calls [createSimpleSyncRestResponseReturnValue](https://github.com/Azure/autorest.java/blob/8ee365815f616d708a129b92610e8e20f24e8b38/javagen/src/main/java/com/azure/autorest/mapper/ClientMethodMapper.java#L1094) to get the return value type, and it has special handling in [processing description](https://github.com/Azure/autorest.java/blob/8ee365815f616d708a129b92610e8e20f24e8b38/javagen/src/main/java/com/azure/autorest/mapper/ClientMethodMapper.java#L1589). It will try to get the description from operation, and put it to the return type description.

But after adoption sdkpackage, I just use `sdkMethod.details ?? ""` to get the description. Since the details is empty(the description is in `summary`), so codegen cannot build the return type description from operation. refer [code](https://github.com/Azure/autorest.java/blob/e2fd61bbaf89359c4cf273de6d1f49de4c3926cc/typespec-extension/src/code-model-builder.ts#L939-L947)

@weidongxu-microsoft , please let me know if this diff is acceptable.

If it's not acceptable, I can think of two solutions:
1. handling in emitter: change `sdkMethod.details ?? ""` to `sdkMethod.details ? sdkMethod.details : sdkMethod.description` when getting operation's description.
2. handling from codegen: use `summary` as well do build the description for return type.

Reply:
I think (2) be better. A merge of "summary" and "description" in Java part, similar to existing logic on model/property/parameter.

Reply:
Fixed in ab74bad4c423c0d37ce3ac929d05413769394b1d